src/main/java/com/forgeessentials/util/CommandParserArgs.java

```diff
@@ -1,14 +1,14 @@
 package com.forgeessentials.util;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Queue;
-import java.util.Set;
-import java.util.TreeSet;
-
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.*;
```

Comment:
Determined what caused it.  Seems like intelej keeps changing imports into a wild card if there is more than 5 from a single package.